packages/grid/_modules_/grid/components/containers/GridRootStyles.ts

```diff
@@ -92,7 +92,7 @@ export const useStyles = makeStyles(
         '& .MuiDataGrid-columnHeader:focus-within, & .MuiDataGrid-cell:focus-within': {
           outline: `solid ${muiStyleAlpha(theme.palette.primary.main, 0.5)} 1px`,
           outlineWidth: 1,
-          outlineOffset: -2,
+          outlineOffset: -1,
```

Comment:
It's off topic, but seems simple enough, it's meant so the outline fits perfectly.

before: 
<img width="423" alt="Capture d’écran 2021-07-20 à 11 48 53" src="https://user-images.githubusercontent.com/3165635/126302363-0cdb35b6-c0b4-4e48-9633-0c68b9b8eb25.png">

after:
<img width="367" alt="Capture d’écran 2021-07-20 à 11 50 17" src="https://user-images.githubusercontent.com/3165635/126302617-da4a6155-2e2c-4064-9b13-3f58d875a758.png">